package.json

```diff
@@ -1,73 +1,53 @@
 {
   "name": "@nuxtjs/redirect-module",
   "version": "0.2.0",
-  "description": "",
+  "description": "Redirect Module makes the redirects will become easier.",
```

Comment:
```suggestion
  "description": "Nuxt module to dynamically redirect initial requests",
```

Maybe?

Reply:
Changed, my english is not very good :worried:

Reply:
@ricardogobbosouza Practice is key! No worries :)